docs/installation.md

```diff
@@ -39,21 +39,18 @@ We do recommend Windows users to use [Docker](#docker) this will work much easie
 - [Microsoft Visual C++ Compiler for Python 2.7](http://www.microsoft.com/en-us/download/details.aspx?id=44266)
 
 ###Easy Installation
-1. Download `PokemonGo-Bot-Install.bat` file from [HERE](https://raw.githubusercontent.com/nivong/PokemonGo-Bot/dev/windows_bat/PokemonGo-Bot-Install.bat)
-2. Run `PokemonGo-Bot-install.bat`
-After that has done the bot will be installed
-3. Run `PokemonGo-Bot-Start.bat`
-This will start the bot and the web interface
+1. Download [PokemonGo-Bot-Install.bat](https://github.com/PokemonGoF/PokemonGo-Bot/blob/master/windows_bat/PokemonGo-Bot-Install.bat)
+2. Download both [PyYAML-3.11-cp27-cp27m-win32.whl](https://github.com/PokemonGoF/PokemonGo-Bot/blob/master/windows_bat/PyYAML-3.11-cp27-cp27m-win32.whl) and  [PyYAML-3.11-cp27-cp27m-win_amd64.whl](https://github.com/PokemonGoF/PokemonGo-Bot/blob/master/windows_bat/PyYAML-3.11-cp27-cp27m-win_amd64.whl).
+3. Download `encrypt.dll`, `encrypt.so` and `encrypt_64.dll`.

```

Comment:
Im gonna make the installer clone the repositor first and then run the correct file.